website/templates/errors/base.html

```diff
@@ -14,8 +14,55 @@
   limitations under the License.
 -->
 
+<style>
+  body {
+    margin: 0;
+    font-family: 'Google Sans', sans-serif;
+  }
+
+  .emblem-message {
+    width: 100%;
+    background-color: #efefef;
+    color: #606060;
+    display: block;
+    padding: 0.75em;
+    text-align: center;
+  }
+  
+  .mdc-top-app-bar, 
+  .mdc-button {
+    background: #306DFD !important;
```

Comment:
Ditto - can we avoid `!important` here?

Reply:
This can get removed 👍🏼

Reply:
As above - keeping `!important` sounds fine, provided we include a link (via code comments) to a GitHub issue tracking its (eventual) removal.